uniffi_bindgen/src/bindings/python/templates/RustBufferTemplate.py

```diff
@@ -16,4 +16,135 @@ def free(self):
         return _UniFFILib.{{ ci.ffi_bytebuffer_free().name() }}(self)
 
     def __str__(self):
-        return "RustBuffer(len={}, data={})".format(self.len, self.data[0:self.len])
\ No newline at end of file
+        return "RustBuffer(len={}, data={})".format(self.len, self.data[0:self.len])
```

Comment:
Note to self: update this to use an explicitly-sized type rather than `ctypes.c_long`.